udata_datalake_service/producer.py

```diff
@@ -0,0 +1,30 @@
+import json
+import os
+
+from kafka import KafkaProducer
+
+KAFKA_HOST = os.environ.get("KAFKA_HOST", "localhost")
+KAFKA_PORT = os.environ.get("KAFKA_PORT", "9092")
+
+
+class KafkaProducerSingleton:
+    __instance = None
+
+    @staticmethod
+    def get_instance() -> KafkaProducer:
+        if KafkaProducerSingleton.__instance is None:
+            KafkaProducerSingleton.__instance = KafkaProducer(
+                bootstrap_servers=f"{KAFKA_HOST}:{KAFKA_PORT}",
+                value_serializer=lambda v: json.dumps(v).encode("utf-8"),
+            )
+        return KafkaProducerSingleton.__instance
+
+
+def produce(key_id, document=None):
+    producer = KafkaProducerSingleton.get_instance()
+    key = key_id.encode("utf-8")
+
+    value = {"service": "datalake", "data": document, "meta": {}}
```

Comment:
can we add at least in message, the format of this resource (known in the kafka consume message).
Because, for further analysis, we would need to filter by type (we will focus first on csv format).